spec/requests/zones_spec.rb

```diff
@@ -25,6 +25,42 @@
   end
 
   context "edit", :edit do
+    it "can create zone authentication" do
+      api_basic_authorize action_identifier(:zones, :edit)
+
+      zone   = FactoryBot.create(:zone, :description => "Current Zone description")
+      params = gen_request(:edit,
+                           "description"               => "Updated Zone Description",
+                           "authentication_attributes" => {"userid" => "foo", "password" => "bar"})
```

Comment:
Should this be `default_authentication` rather than `authentication_attributes`?

Reply:
Again, this is a change that was introduced as a part of https://github.com/ManageIQ/manageiq/pull/21047 so `authentication_attributes` is from the `allow_nested_attributes_for` change made there.